docker/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM phusion/baseimage:0.10.1
+LABEL maintainer "chevdor@gmail.com"
+
+RUN apt-get update && \
```

Comment:
I'm not into docker stuff, but maybe it would be better to split this into separate steps? I found it more useful since if you change some later steps, previous steps could be reused.

Reply:
That makes it more readable and I would do that also in a regular script but for docker this is different. Having a 'single' command packs all in a single layer.

That could be split however but I have very bad internet atm and testing the build of docker images is a pain :)

Reply:
If you split `apt-get update` from `upgrade` and `install`, then docker will cache the `update` step and then when you try to build the rest of the image it may fail because the outcome of the `update` is outdated.